downloads/platform.md

```diff
@@ -181,14 +181,11 @@ Recent Debian/ubuntu distributions include their own build of Julia, which can b
 sudo apt install julia
 ```
 
-## Arch Linux
-The Arch User Repository has [a package for Julia](https://aur.archlinux.org/packages/julia-bin) that is built from the official binaries of Julia. To install it run:
+## Arch Linux and Manjaro
+The Arch Community Repository has [a package for Julia](https://archlinux.org/packages/community/x86_64/julia/) to allow for installation using pacman. To install it run:
```

Comment:
This is a different package, which we _don't_ want to encourage users to install